internal/mqttconf/package.go

```diff
@@ -0,0 +1,2 @@
+// Package mqtt contains supporting utils shared between MQTT reader and writer.
+package mqttconf
```

Comment:
Not 100% sure, but I think an empty file like this may not be required (?)

Reply:
I think they exist to hold the package-level documentation? All benthos packages seems to have them.